.gitignore

```diff
@@ -0,0 +1,7 @@
+./node_modules
```

Comment:
We need to remove the ./ from the directory since it;'s on the main path.

Reply:
They refer to the same path even with `./` but you're alright it make more sense to be removed since it doesn't add anything. I will commit the changes.

Reply:
I was thinking the same but I see that on some systems it's tracking the directory when the `./` is presented.
On my Windows Linux Subsystem, it's tracking `node_modules`

Reply:
Nice to know! Thanks for pointing me out to this. I've committed the change